react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx

```diff
@@ -221,7 +221,7 @@ class LocalVideoMenuTriggerButton extends Component<IProps> {
                     overflowDrawer = { _overflowDrawer }
                     position = { _menuPosition }
                     visible = { popoverVisible }>
-                    {!_overflowDrawer && buttonVisible && !isMobileBrowser() && (
+                    {buttonVisible && !isMobileBrowser() && (
```

Comment:
doesn't this need the narrow check?

Reply:
for mobile the trigger for menu is long press; for desktop even when we have narrow(drawer menu) we still want the trigger for the menu to be the buttons.

Reply:
Gotcha!